cves/kernel/CVE-2019-15666.yml

```diff
@@ -305,9 +310,9 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: There was no discussion around this issue
```

Comment:
Some (but not really) discussions here: https://lore.kernel.org/all/20190228075226.hza3hydllddj65nd@gondor.apana.org.au/T/, https://bugzilla.redhat.com/show_bug.cgi?id=1747334. As mentioned above, someone wrote a POC